conversions/weight_conversion.py

```diff
@@ -0,0 +1,64 @@
+"""
+Functions useful for conversion fo weight units
+"""
+
+
+def weightConversion(from_type: str, to_type: str, value: float) -> any:
```

Comment:
```suggestion
def weight_conversion(from_type: str, to_type: str, value: float) -> any:
```
Python naming convention